src/py_converters_11.h

```diff
@@ -113,14 +177,14 @@ namespace PYBIND11_NAMESPACE { namespace detail {
                 return true;
             }
 
-            auto vertices = src.attr("vertices");
-            auto codes = src.attr("codes");
+            py::object vertices = src.attr("vertices");
+            py::object codes = src.attr("codes");
             auto should_simplify = src.attr("should_simplify").cast<bool>();
             auto simplify_threshold = src.attr("simplify_threshold").cast<double>();
 
-            if (!value.set(vertices.ptr(), codes.ptr(),
+            if (!value.set(vertices.inc_ref().ptr(), codes.inc_ref().ptr(),
                            should_simplify, simplify_threshold)) {
-                return false;
+                throw py::error_already_set();
```

Comment:
This was technically wrong before, but otherwise, `pybind11` would substitute its own generic "cannot convert type PyType to C++Type" error message, so not fatal.